src/OrchardCore.Modules/OrchardCore.ContentFields/Views/DateField-Custom.Edit.cshtml

```diff
@@ -0,0 +1,37 @@
+@model OrchardCore.ContentFields.ViewModels.EditDateFieldViewModel
+
+@using OrchardCore.ContentFields.Settings;
+@using OrchardCore.ContentManagement.Metadata.Models
+@using OrchardCore.ResourceManagement
+
+@inject IResourceManager ResourceManager
+
+@{
+    var settings = Model.PartFieldDefinition.Settings.ToObject<DateFieldSettings>();
+    AppendDatePickerScript();
+}
+
+@functions{
+    private void AppendDatePickerScript()
+    {
+        const string cultureSeparator = "-";
+        var culture = Orchard.CultureName();
+        var language = culture.Contains(cultureSeparator)
+            ? culture.Substring(0, culture.IndexOf(cultureSeparator))
+            : culture;
+        var datePickerScript = new TagBuilder("script");
+        datePickerScript.InnerHtml.AppendHtml($@"$(function () {{
+            $('.datepicker').datepicker($.datepicker.regional['{language}']);
+        }});");
+
+        ResourceManager.RegisterFootScript(datePickerScript);
```

Comment:
Is there a way to register this one with a name such that if multiple editors are added to the same page, it will only register this block once?

Reply:
My aim to register this once